src/components/MultiChoice/index.js

```diff
@@ -0,0 +1,78 @@
+// TODO INSERT A COMMENT REFERENCE TO EXTERNAL URL IF POSSIBLE
+
+import React from "react";
+import PropTypes from "prop-types";
+import glamorous from "glamorous";
+import { ERROR_COLOUR } from "govuk-colours";
+
+import LabelText from "../LabelText/index";
+import ErrorText from "../ErrorText/index";
+import HintText from "../HintText/index";
+import {
+  BORDER_WIDTH_MOBILE,
+  MEDIA_QUERIES,
+  SITE_WIDTH,
+  SPACING
+} from "../../constants/index";
+
+const FieldSet = glamorous.div(
+  {
+    padding: 0,
+    margin: 0,
+    border: 0,
+    boxSizing: "border-box",
+    width: "100%",
+    ":after": {
+      content: "''",
```

Comment:
Same here, why `"''"`, did you mean `""`?

Reply:
in CSS it is: `content: "";` (to have empty content). In CSS-in-JS you need to have single quotes inside the `content` property

Reply:
oh right ha